### PR TITLE
Updating dependencies; addresses #38

### PR DESCRIPTION
**Pull request by mathew-kurian**
#38

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -18,19 +18,21 @@
...
+    "jss-default-unit": "*",
+    "jss-nested": "^2.5.0",
+    "jss-vendor-prefixer": "^3.0.1",
+    "keycode": "^2.1.7",
+    "lodash": "^4.16.6",
+    "narcissus": "^1.0.0",
     "react-center-component": "^3.0.0",
-    "react-jss": "^2.0.6"
+    "react-jss": "^4.1.3",
+    "update": "^0.7.4"
```
Comment (qimingweng) [inline package.json:35]: What's this?

File: src/CloseCircle.js [hunk 2]
```diff
@@ -4,33 +4,35 @@ import React, {PropTypes} from 'react';
 const RECT_WIDTH = 1.5;
 const MARGIN = 8;
 
-const CloseCircle = (props) => {
-  const {
-    diameter,
-  } = props;
+class CloseCircle extends React.PureComponent {
```
Comment (qimingweng) [inline src/CloseCircle.js:7]: What's the benefit of using PureComponent (over a function? I mean I guess are functional components not treated as implicitly pure?) and which version of react was this first introduced in?

**Comment by qimingweng** [comment]
I have addressed the dependencies here 7c4a2b2af42b09462f91e7185248abb7d7dccf42, some of the other changes let's talk more in an issue or another PR?

File: src/CloseCircle.js (hunk 2 above)
Comment (mathew-kurian) [reply src/CloseCircle.js:7]: Functional components are actually a better choice. This was the result of a hasty solution to make the library work on Node 6.

File: package.json (hunk 1 above)
Comment (mathew-kurian) [reply package.json:35]: The last module is not needed.

**Comment by mathew-kurian** [comment]
Ok great. Your latest push should do the trick. This PR was merely to get the ball rolling. This a great module btw!

File: src/CloseCircle.js (hunk 2 above)
Comment (qimingweng) [reply src/CloseCircle.js:7]: Why was it breaking on Node 6? We run the package on Node 6 just fine

**Comment by mathew-kurian** [comment]
Perhaps it was a temporary dep caching issue from registry. One of the JSS packages was giving an issue. It enforced jss version < 4 which then prevented the module from running on Node 6

**Pull request closed without merging**